input/tex/eqnumbers.rst

```diff
@@ -19,7 +19,7 @@ automatic equation numbers by adding:
 
 to tell the TeX input processor to use the AMS numbering rules (where
 only certain environments produce numbered equations, as they would be
-in LaTeX).  It is also possible to set the tagging to `'all'`, so that
+in LaTeX).  It is also possible to set the tagging to ``all``, so that
```

Comment:
It actually should be ``` ``'all'`` ``` to be consistent with other usage (I am so used to using single back-ticks here on GitHub that I sometimes get it wrong in `.rst` files, which require double-backtics).

I'll take care of it (it needs to go into `v3.1-latest` first, then that is merged into `master`).